infrastructure/kubernetes/common/README.md

```diff
@@ -0,0 +1,49 @@
+# Kubernetes: mezo-staging/monitoring
+
+This module contains Kubernetes deployments for monitoring the mezo staging nodes.
+
+### Prerequisites
+
+- Infrastructure components for the `mezo-<environment>` GCP project created using the
+  corresponding Terraform module (make sure GKE version is >=1.29)
+- `gcloud` installed and authorized to access the `mezo-<environment>` GCP project
+- `kubectl` tool installed
+
+### Secrets
+
+#### Grafana
+
+Secrets are used to set the default user and password to log in to
+the Grafana UI. Here are the required entries:
+- admin-password
+- admin-user
+
+Use `kubectl` to create the secrets or apply changes:
+```Shell
+kubectl create secret generic -n monitoring grafana-secret \
+  --from-literal=admin-user=<STAGING_USER> \
+  --from-literal=admin-password=<STAGING_PASSWORD>
+```
+
+#### Metrics Scraper
+
+One secret is used to set the configuration of the metrics scraper service. To
+create it, use the following command:
+```Shell
+kubectl create secret generic metrics-scraper-config -n monitoring \
+  --from-file=config.json=<PATH_TO_CONFIG>
```

Comment:
Let's add a short example of the json file here. Moreover, let's reference `metrics-scraper/config.go` as the schema.

Reply:
addressed indeed: 0041b6d8c5e78e26f2651cd8d2ec085728b67a0e